operator/gitops/argocd/pipeline-service/tekton-results/api-kube-rbac-proxy.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tekton-results-api
+  namespace: tekton-pipelines
+spec:
+  template:
+    spec:
+      containers:
+        - name: kube-rbac-proxy
+          image: registry.redhat.io/openshift4/ose-kube-rbac-proxy:v4.12
+          args:
+            - "--secure-listen-address=0.0.0.0:9443"
+            - "--upstream=http://127.0.0.1:9090/"
+            - "--logtostderr=true"
+            - "--v=10"
```

Comment:
this is too high a logging verbosity level for staging / prod

Reply:
v2 or v4 instead would be ideal

Reply:
Agree. In sprayproxy also v10 is producing to much information in the logs with complete request/response logging.

Reply:
Sure, I can make that change. Initially, it was good for debugging purposes but I understand the concern.

Reply:
I did an investigation on various verbosity levels. And I found out there's almost no logs (no log for a new made) for a verbosity level less than equal to 5. 
The best I can bring this down to while having some sort information when a new request is made,  is 6.
The logs with verbosity - 6
![Screenshot from 2023-07-28 13-36-07](https://github.com/openshift-pipelines/pipeline-service/assets/54385449/048ba831-8337-4e7d-8333-75fb71c4d614)